services/src/main/java/org/jboss/windup/web/services/rest/WindupEndpointImpl.java

```diff
@@ -55,18 +65,49 @@
     private Queue executorQueue;
 
     /**
+     * To avoid problems with passing detached entities,
+     * load related entities during cloning
+     */
+    protected AnalysisContext cloneExecution(AnalysisContext originalContext)
+    {
+        Set<Package> allPackages = new HashSet<>();
+
+        allPackages.addAll(originalContext.getIncludePackages());
+        allPackages.addAll(originalContext.getExcludePackages());
+
+        List<Long> packageIds = allPackages.stream().map(aPackage -> aPackage.getId())
+                .collect(Collectors.toList());
+
+        AnalysisContext clone = originalContext.clone();
```

Comment:
Hmm, it seems like it should be possible to just call entityManager.merge? It may be that the CascadeType.MERGE needs to be added to the relationship as well (if it isn't already).